README.md

```diff
@@ -128,26 +129,65 @@ The below commands can be used to test payer account credentials where we create
 2. `aws organizations list-accounts --profile=orgtest | jq '.[][] | select(.Name=="username-cli-test")'`
 3. `aws sts assume-role --role-arn arn:aws:iam::<ID>:role/OrganizationAccountAccessRole --role-session-name username-cli-test --profile=orgtest`
 
-### 1.5. Local Development
+### 1.5. Development
+
+It is recommended to let the operator know when you're running it for testing purposes.
+This has benefits such as skipping AWS support case creation.
+This is done by setting `FORCE_DEV_MODE` in the operator's environment.
+This is handled for you if you use one of the `make deploy-*` targets described below.
 
 ### 1.5.1 Operator Install
 
-The operator can be installed locally on a Minishift or Code-Ready-Containers (CRC) cluster (or a private OpenShift cluster).  On a new local cluster, running the Makefile `deploy` target (passing the Access Key ID and Secret Access Key) will install the operator, and create all the necessary OpenShift CRDs and secrets for it to work locally.
+The operator can be installed into various cluster and pseudo-cluster environments. Depending which you choose, you can run in "local" mode or in "cluster" mode. The former is known to work in a Minishift or Code-Ready-Containers (CRC) cluster, or a private OpenShift cluster. The latter is known to work in a real OSD cluster. You can try to mix and match; it might work.
```

Comment:
What is a `private OpenShift cluster` in this context?

Reply:
No idea, just kept that from the previous version.